src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/RequestForReconsiderationDeletionDefendantNotificationHandler.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.defendant;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.callback.CaseProgressionDashboardCallbackHandler;
+import uk.gov.hmcts.reform.civil.client.DashboardApiClient;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.DashboardNotificationsParamsMapper;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
+
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.DELETE_NOTIFICATION_REQUEST_FOR_RECONSIDERATION_DEFENDANT;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA6_CP_SDO_MADE_BY_LA_DELETE_DEFENDANT;
+
+@Service
+public class RequestForReconsiderationDeletionDefendantNotificationHandler extends CaseProgressionDashboardCallbackHandler {
```

Comment:
Is this a temporary handler until CIV-13715 is through? Or is it doing something else?

Reply:
Yes, it is a temporary handler that only removes the notification. Later, it will be deleted inside the db migration script that generates the following notification